docs/source/markdown/podman-machine-inspect.1.md

```diff
@@ -0,0 +1,35 @@
+% podman-machine-inspect(1)
+
+## NAME
+podman\-machine\-inspect - Inspect one or more virtual machines
+
+## SYNOPSIS
+**podman machine inspect** [*name*]...
+
+## DESCRIPTION
+
+Inspect one or more virtual machines
+
+Obtain greater detail about podman virtual machines.  More than one virtual machine can be
+inspected at once.
+
+## OPTIONS
+#### **--format**
+
+Print results with a golang template.
```

Comment:
```suggestion
Print results with a Go template.
```
Go is the proper name, but "Golang" is commonly used.  If you still want golang, please cap the G.